psi4/src/psi4/dcft/dcft_compute.cc

```diff
@@ -47,6 +48,18 @@ namespace dcft {
 double DCFTSolver::compute_energy() {
     double total_energy = 0.0;
 
+    if (options_.get_str("DCFT_GUESS") == "DCFT") {
+        // We're reusing DCFT files from a previous computation. Let's check that one exists.
+        if (!((same_a_b_orbs_ == true && psio_tocentry_exists(PSIF_DCFT_DPD, "Lambda SF <OO|VV>")) ||
+              (same_a_b_orbs_ == false && psio_tocentry_exists(PSIF_DCFT_DPD, "Lambda <Oo|Vv>")))) {
+            throw PSIEXCEPTION("Could not find a previous DCFT computation as the DCFT_GUESS=DCFT guess.");
+        }
+    } else {
+        // Obliviate all old DCFT files.
+        psio_->close(PSIF_DCFT_DPD, 0);
+        psio_->open(PSIF_DCFT_DPD, PSIO_OPEN_OLD);
```

Comment:
I'm not sure how this works. If the `close( ..., 0)` should remove the file, then `open( ..., ` should be on a file that is new, not old. Or `close(..., 0)` truncates to 0 instead of removing? If the latter is true, then the following should be corrected:

https://github.com/psi4/psi4/blob/fbb2ff444490bf6b43cb6e027637d8fd857adcee/psi4/src/psi4/libpsio/psio.hpp#L229-L230

Reply:
If I had to guess, `psio_->open` silently creates the file if it's been removed.

Making `libpsio` comprehensible to mere mortals is far, far, far outside the scope of this PR.